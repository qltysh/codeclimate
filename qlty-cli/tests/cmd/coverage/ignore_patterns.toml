args = [
  "coverage",
  "publish",
  "--dry-run",
  "--override-commit-sha",
  "822641a8a442eab5981096a0d734ac180aa40106",
  "--override-build-id",
  "123",
]
bin.name = "qlty"

[env]
remove = ["GITHUB_ACTIONS", "RUST_BACKTRACE"]

[env.add]
QLTY_COVERAGE_TOKEN = "123"
