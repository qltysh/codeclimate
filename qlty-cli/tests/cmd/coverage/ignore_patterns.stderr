qlty [..] ([..] debug)
https://qlty.sh/d/coverage

  Retrieving CI metadata...
  → unknown CI commit "822641a8a442eab5981096a0d734ac180aa40106" on branch ""

  Reading code coverage data...
  → Found 1 files with code coverage data

  Exporting code coverage data...
  → Exported to "tmp/qlty-coverage"
